wellness-bet-sleep-frontend/src/Components/dashboard/VenmoUpload.js

```diff
@@ -15,58 +15,84 @@ class VenmoUpload extends Component {
+      "state_changed",
+      snapshot => {},
       error => {
-          alert(error);
+        alert(error);
       },
       () => {
-          storage.ref('images').child(currentVenmoName).getDownloadURL().then(url => {
-              console.log(url);
-              this.setState({
-                  venmoPhoto: url
-              });
+        storage
+          .ref("images")
+          .child(currentVenmoName)
+          .getDownloadURL()
+          .then(url => {
+            console.log(url);
+            this.setState({
+              venmoPhoto: url
+            });
 
-              // store venmo screenshot in the database
+            // store venmo screenshot in the database
 
-              const participantScreenshot = {
-                  venmoPhoto: url,
-                  groupId: this.props.joinCode,
-                  partUserId: this.props.partUserId
-              };
+            const participantScreenshot = {
+              venmoPhoto: url
+            };
 
-              axios.post(`/api/participant/add`, {...participantScreenshot}).then(res => {
-                  console.log(res.data);
-              }).catch(error => console.log(error))
-;          })
-      })
-  } 
+            // const groupId = this.props.joinCode;
+            // const partUserId = this.props.partUserId
+            console.log(participantScreenshot);
+
+            axios
+              .put(`/api/participant/${this.props.partUserId}`, {
+                ...participantScreenshot
+              })
+              .then(res => {
+                console.log(res.data);
+              })
+              .catch(error => console.log(error));
+          });
+      }
+    );
+  };
   render() {
     return (
-        <div>
-           <h1>Upload Venmo screenshot for proof of payment here</h1>        
+      <div style={{ marginTop: "100px" }}>
+        <h1>Upload Venmo screenshot for proof of payment here</h1>
         <form onSubmit={this.uploadVenmo}>
-        <input type="file" className="process__upload-btn" onChange={(e) => this.uploadVenmo(e)} />
-        <img src={this.state.venmoPhoto} alt="venmo screenshot" className="process__image" />
-        
+          <input
+            type="file"
+            className="process__upload-btn"
+            onChange={e => this.uploadVenmo(e)}
+          />
+          <img
+            src={this.state.venmoPhoto}
+            alt="venmo screenshot"
+            className="process__image"
+          />
         </form>
-       
-        </div>
-        )
+      </div>
+    );
   }
 }
 
 const mapStateToProps = state => {
-    console.log(state.groups.groups, 'state from venmo upload')
-    return {
-      joinCode: state.groups.groups.joinCode,
-      partUserId: state.auth.user.firebase_id
-    }
-  }
+  console.log(state.groups.groups, "state from venmo upload");
+  return {
+    joinCode: state.groups.groups.joinCode,
+    partUserId: state.auth.user.firebase_id,
+//     participantId: state.auth.user.id
```

Comment:
removed reference to participant id, since user.id doesn't exist. only firebase id